daisy_workflows/import_userguide.md

```diff
@@ -0,0 +1,275 @@
+# Importing Virtual Disks into Google Compute Engine (GCE)
+
+
+# Introduction
+
+Daisy is an open source workflow tool for building custom GCE images and importing/exporting to/from virtual disks.  This document provides instructions for using Daisy to import VMware VMDK and other Virtual Disks.
+
+Some basic concepts to start with
+
+*   **Virtual Disk**: Virtual disk is a file that encapsulates the content of a virtualized disk in a virtualization environment. Virtual Disks are critical components of virtual machines for holding boot media and data. Virtualization platforms (eg VMWare, Hyper-v, KVM etc) each have their own format for virtual disks.
+*   **Persistent Disk**: Compute Engine Persistent Disk is a Compute Engine resource that is equivalent to disk drives in physical computers and virtualdisks in a virtualization environment. 
+*   **GCE Image**: Image is an immutable representation of a Persistent Disk and is used for creating multiple disks from one single templatized version.
+
+# Installation
+
+It is recommended that you run Daisy in a GCE VM instance that is configured with required API scopes and workflow files pre installed. Executing the following command will create a new VM instance and install required files.
+
+```
+gcloud compute instances create daisy-control --image-project debian-cloud --image-family debian-9 --scopes=compute-rw,storage-full --metadata startup-script-url=gs://compute-image-tools/daisy_import_alpha.sh
+```
+
+# Importing Virtual Disks using Daisy
+
+### Step 1: Upload Virtual Disk to Google Cloud Storage
+
+Upload your virtual disk files to Google Cloud Storage (GCS) by following instructions at https://cloud.google.com/storage/docs/object-basics
+
+### Step 2: Connect to the VM instance that has Daisy installed
+
+Connect to the VM that was created in the Installation section. You can connect to the VM by running the following command.
+
+```
+gcloud compute ssh daisy-control
+```
+
+### Step 3: Convert Virtual Disk to Compute Engine Image
+
+Now inside the VM,run the workflow titled import_image.wf.json to convert the virtual disk file in GCS to a Compute Engine Image.
+
+```
+daisy -variables source_disk_file=YOUR_VIRTUAL_DISK_FILE \\
```

Comment:
do these line continuations work with daisy? seems like parsing the args would fail as these variables need to be one arg